templates/home.html

```diff
@@ -2,16 +2,110 @@
 {% block header_css %}
   <head>
       <title>Game Room Tracker</title>
+      <link href="/static/select2-4.0.3/select2.min.css" rel="stylesheet" />
   </head>
   {{ super() }}
 {% endblock %}
 {% block body %}
   <div class="row">
     <div class="col-md-12 col-sm-12">
-      <h4>Game Room Tracker</h4>
+      <h3>Game Room Tracker</h3>
+    </div>
+  </div>
+  <hr />
+  <div class="row">
+    <div class="col-md-12 col-sm-12">
+      <h4>Create a new game</h4>
+    </div>
+  </div>
+  <div class="row">
+    <div class='col-md-4 col-sm-4'>
+      <select class="form-control" id="game-select">
+        <option></option>
```

Comment:
this might be overkill for this mvp, but how do you feel about passing the options in from the backend and iterating through them here?